llama_index/indices/vector_store/base.py

```diff
@@ -80,7 +83,7 @@ def as_retriever(self, **kwargs: Any) -> BaseRetriever:
         )
 
     def _get_node_embedding_results(
```

Comment:
curious: if index object already keeps track of `show_progress` as an attribute, why pass it here again?

Reply:
We only want the progress bars to appear when the user is initially creating an index, rather than every time a new document is added to the index. `_build_index_from_nodes` is the target function that indicates the creation of the index, but some of the functions it may call may be used in other non-index creation operations.